package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "platformicons",
-  "version": "3.5.2",
+  "version": "3.5.3",
```

Comment:
https://www.npmjs.com/package/platformicons

Seems 3.5.2 is still the latest, maybe we revert this?